packages/dappmanager/src/modules/installer/getInstallerPackageData.ts

```diff
@@ -10,18 +14,21 @@ export function getInstallerPackagesData({
   releases,
   userSettings,
   currentVersions,
-  reqName
+  reqName,
+  packagesInfo
 }: {
   releases: PackageRelease[];
   userSettings: UserSettingsAllDnps;
   currentVersions: { [dnpName: string]: string | undefined };
   reqName: string;
+  packagesInfo: InstalledPackageData[];
 }): InstallPackageData[] {
   const packagesDataUnordered = releases.map(release =>
     getInstallerPackageData(
       release,
       userSettings[release.dnpName],
-      currentVersions[release.dnpName]
+      currentVersions[release.dnpName],
+      packagesInfo.find(pkg => pkg.dnpName === release.dnpName)
```

Comment:
If you only need one package there's no need to list all packages in `packages/dappmanager/src/calls/packageInstall.ts`